.github/workflows/post-integration.yaml

```diff
@@ -0,0 +1,58 @@
+# This workflow is resposible for all kinds of integration tests that are run AFTER MERGE TO MAIN
+# Those tests depend on a container image so in the first job we wait for the image build to succeed
+
+name: Post submit integration tests
```

Comment:
Maybe we can just name them main integration tests, since this "post submit" convention is introduced by Prow. We can then also name the file `main-Integration.yaml`.